### ISSUE TITLE
amp-selector[multiple] should display cursor: pointer on selectable options

### ISSUE THREAD

**Issue by caroqliu** [ampproject/amphtml#28308]
> In multiselect, `cursor:pointer` should be on. Or more generically, if we have a meaningful action attached to the click (e.g. unselect for multiselect items), cursor should be pointer.

[Related comment thread](https://github.com/ampproject/amphtml/pull/27365#discussion_r420203420)
Caveat: Unless item is `disabled`

**Comment by THE-VR7** [comment]
Hello. I am new  and would love to contribute.
I would like to fix this issue, if no one is already on it.

**Comment by caroqliu** [comment]
Sure thing, thanks for your contribution!

**Comment by AtharvaSune** [comment]
Hi, I am new here. If the issue is not already assigned I would like to work on it

**Comment by caroqliu** [comment]
@THE-VR7 Let us know if you are still planning to contribute. Otherwise it sounds like @AtharvaSune will be happy to take this on. :) No rush either way.

**Comment by THE-VR7** [comment]
yeah sure @AtharvaSune  can work on it, I am also new here and i am already working on another issue of this repository, so no problem

**Comment by AtharvaSune** [comment]
hey @caroqliu , from what i understood from the linked thread in the description, i need to add 
```
amp-selector[multiple] [option][selected] {
  cursor: pointer;
  outline: solid 1px rgba(0, 0, 0, 0.7);
}
```
something like this in the extensions/amp-selector/0.1/amp-selector.css, right ?

**Comment by caroqliu** [comment]
@AtharvaSune Yes that's right :)

**Comment by AtharvaSune** [comment]
Hey @caroqliu just submitted the pr, could you have a look at it , :sweat_smile:  if any further changes are required do tell
